content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Drivers'
+metaTitle: 'Drivers (Preview)'
+metaDescription: 'Learn how you can use Prisma with other database drivers using driver adapters (Preview)'
+preview: true
+tocDepth: 3
+---
+
+## Built-in drivers (default)
+
+One of Prisma Client's component's is the [Query Engine](./prisma-engines/query-engine) <span class="concept"></span>. The Query Engine transforms Prisma Client queries to SQL statements that are executed using the included database driver. The Query Engine also includes built-in database drivers that don't require any additional effort to set up. The database drivers use TCP connections to connect to the database.
+
+![Query flow from the user application to the database with Prisma Client](./images/drivers/qe-query-execution-flow.png)
+
+## Driver adapters
+
+Prisma Client can connect and run queries against your database using other JavaScript database drivers using **driver adapters**. Adapters act as _translators_ between Prisma Client and the JavaScript database driver.
+
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+
+Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
+
+Prisma maintains the following driver adapters:
+
+- Serverless database driver adapters:
+  - [Neon](#neon)
+  - [PlanetScale](#planetscale)
+- Database driver adapters:
```

Comment:
This is more of a question: What makes Turso _not_ a serverless database driver? Do we need to make this distinction?

Reply:
Okay. I agree that it's different but I don't think I agree that we should separate it here. But it's ok either way.

Reply:
If we don't want to separate, we can't give it the grouping "Serverless database driver" as Turso is not that.

Reply:
I think we are saying the same thing: My suggestion is "Prisma maintains the following driver adapters:" followed by the adapters we maintain. I don't understand why we are specifying further here.